README.md

```diff
@@ -9,6 +9,14 @@ A Config Controller
 Using YAML files *endpoints* can be defined to be exposed or called on specified conditions. The call of an endpoint results in *coco* calling the same endpoint on a given set of hosts. *coco* can be configured to check or compare the results and based on that export [prometheus](prometheus.io) alerts, write [slack](slack.com) messages or call other endpoints.  
 The values passed to *endpoints* can be marked as part of a global state, which is tracked and kept in sync on the hosts.
 
+## Metrics
```

Comment:
I think this should go to the rst documentation. But since that is still empty and unstructured (even though it exists!), let's keep it here and open an issue for it to get moved. 
Great you documented this!

Reply:
#70